launch.go

```diff
@@ -171,6 +195,24 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath string) error
 		return fmt.Errorf("fetching Pipeline ID %d: %v", j.PipelineID, err)
 	}
 
+	log.Printf("Fetching Parent Build %d for Meta", b.ParentBuildID)
```

Comment:
The first job `main` and `pr` jobs will not have the `parentBuild Id`. But we shouldn't treat them as errors. should probably just skip fetching parent meta in those cases.